src/TestInterfaces/TestInterfaces.csproj

```diff
@@ -0,0 +1,54 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We need another project to test that one can indeed derive from a C# interface in an external assembly.